tests/lsp_features/semantic_tokens.zig

```diff
@@ -1960,6 +1960,66 @@ test "weird code" {
     , &.{}, .{ .mode = .zon });
 }
 
+test "mutable" {
```

Comment:
These tests don't cover anything that isn't already covered by the existing tests. They should be removed.